documentation/mqtt-overview.md

```diff
@@ -0,0 +1,83 @@
+# AnkerMake MQTT Communications Overview
+
+We're going over the AnkerMake implementation of the [MQTT communications standard](https://mqtt.org/). While we won't be explaining the MQTT standard itself, we will explain certain aspects of how the protocol works for ease of understanding.
+
+## Generic MQTT Basics
+
+MQTT (Messaging Que Transport Telemetry) is a [stateful](https://byjus.com/gate/difference-between-stateless-and-stateful-protocol/#:~:text=What%20is%20a%20Stateful%20Protocol,Protocol%20(FTP)%2C%20etc.), [edge driven](https://en.wikipedia.org/wiki/Edge_computing), lightweight, and low bandwidth communications protocol primarily used by [IoT devices](https://en.wikipedia.org/wiki/Internet_of_things). It was specifically designed for industry, so that's the context in which we'll lay out our brief explanation.
```

Comment:
I'm about 0% convinced that Anker's use of MQTT constitutes "Edge driven"/"Edge computing". That is a buzzword-heavy term, with little technical backing, and I'd prefer if we dropped it from this guide.

Same for "stateful". Depending on how you look at it, MQTT is *not* stateful. Rather it's an event transport mechanism. Any number of publishers and consumers can each update and be updated about subscribed topics.